### ISSUE TITLE
Ajax requests are not considered

### ISSUE THREAD

**Issue by fduarte42** [php-middleware/phpdebugbar#41]
When using ajax requests these requests are not integrated into debugbar, because they are not considered.

This Pullrequest fixes this ( [PR 40](https://github.com/php-middleware/phpdebugbar/pull/40) ).

Do you have a timeframe when this will be merged and released?

**Comment by tyrsson** [comment]
Its rendered in my application, the bars are stacked one on top of the other so you end up with one for the original request, then it stacks one for each ajax request or at least that is the behavior I was getting.

**Comment by fduarte42** [comment]
Yes, correct. But this is faulty and this PR fixes that. As the debugbar module can handle ajax requests, but this bridge does not use that funcionality. The PR adds that needed parameters.

**Comment by tyrsson** [comment]
[offtopic]
I thought it was ironic we used the same exact variable name: isAjax.
[/offtopic]

**Comment by fduarte42** [comment]
The PR is not from me, but yes I did notice that. ;-)
